### PR TITLE
[MISC] Clarify: FieldMap PE technique --> no SPM

### PR DESCRIPTION
**Pull request by CPernet**
SPM doesn't deal with that case

### CODE REVIEW

**Review by Remi-Gau — APPROVED** [review]
AFAIK you are correct @CPernet.

@gllmflndn can you confirm?

If you need context: 

https://bids-specification.readthedocs.io/en/latest/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#case-4-multiple-phase-encoded-directions-pepolar

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 1]
```diff
@@ -1228,7 +1228,7 @@ map corresponding to the nonuniformities of the *B<sub>0</sub>* field.
 These `*_epi.nii[.gz]` - or `*_m0scan.nii[.gz]` for arterial spin labeling perfusion data - files can be 3D or 4D --
 in the latter case, all timepoints share the same scanning parameters.
 Examples of software tools using these kinds of images are FSL TOPUP,
```
Comment (tsalo) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md:1230]:
```suggestion
Examples of software tools using these kinds of images are FSL TOPUP and
```

**Review by effigies — APPROVED** [review]
Seems no *less* correct, regardless of what SPM can do.

**Comment by sappelhoff** [comment]
Thanks @CPernet

**Pull request merged**